src/sentry/tsdb/redis.py

```diff
@@ -485,16 +485,17 @@ def get_frequency_series(self, model, items, start, end=None, rollup=None):
 
         # Freeze ordering of the members (we'll need these later.)
         for key, members in items.items():
-            items[key] = tuple(members)
+            items[key] = list(members)
```

Comment:
Yeah, the comment should have probably been more specific — it is pretty ambiguous and I can see where the confusion comes from.